scaleway/resource_lb_route.go

```diff
@@ -39,10 +39,28 @@ func resourceScalewayLbRoute() *schema.Resource {
 				ValidateFunc: validationUUIDorUUIDWithLocality(),
 				Description:  "The backend ID destination of redirection",
 			},
-			"match_sni": {
-				Type:        schema.TypeString,
+			"match": {
```

Comment:
This is a breaking change for the schema. Please add a new version and a the methods to upgrade the test.

Reply:
Made changes on schema that avoid the breaking change